features/modules-crud.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Check that user is able to perform create, read, update and delete operations in api modules
```

Comment:
All modules should be in the separate files
for each module better to create separated steps and features